src/frontend/sharedComponents/OfflineAreaCard.tsx

```diff
@@ -0,0 +1,161 @@
+import * as React from "react";
+import { defineMessages, FormattedMessage, useIntl } from "react-intl";
+import { StyleSheet, View } from "react-native";
+import { LIGHT_GREY } from "../lib/styles";
+import Text from "./Text";
+
+//Lev
+
+const m = defineMessages({
+  lvlDetail: {
+    id: "sharedComponent.OfflineAreaCard.lvlDetail",
+    defaultMessage: "Level of Detail",
+  },
+  zoomLevel: {
+    id: "sharedComponent.OfflineAreaCard.zoomLevel",
+    defaultMessage: "Zoom Level",
+  },
+  lvlDetailGlobal: {
+    id: "sharedComponent.OfflineAreaCard.lvlDetailGlobal",
+    defaultMessage: "Global",
+    description: "Level of detail seen on map - farthest zoom level",
+  },
+  lvlDetailSubcontinent: {
+    id: "sharedComponent.OfflineAreaCard.lvlDetailSubcontinent",
+    defaultMessage: "Subcontinent",
+    description: "Level of detail seen on map - second farthest zoom level",
+  },
+  lvlDetailLargeCountry: {
+    id: "sharedComponent.OfflineAreaCard.lvlDetailLargeCountry",
+    defaultMessage: "Large Country",
+    description: "Level of detail seen on map - can see large countries",
+  },
+  lvlDetailSmallCountry: {
+    id: "sharedComponent.OfflineAreaCard.lvlDetailSmallCountry",
+    defaultMessage: "Small Country",
+    description: "Level of detail seen on map - can see smaller countries",
+  },
+  lvlDetailLargeMetro: {
+    id: "sharedComponent.OfflineAreaCard.lvlDetailLargeMetro",
+    defaultMessage: "Large Metropolitan Area",
+    description:
+      "Level of detail seen on map - Can see details of large metropolitan areas",
```

Comment:
for the sake of consistency: 

```suggestion
      "Level of detail seen on map - can see details of large metropolitan areas",
```